native/cocos/scene/RenderScene.h

```diff
@@ -85,7 +85,6 @@ class RenderScene : public RefCounted {
     void removeSpotLights();
 
     void addModel(Model *);
-    void removeModel(index_t idx);
```

Comment:
Why remove it?

Reply:
The function doesn't exist in TS and it's not used too.